src/components/Home.tsx

```diff
@@ -1,14 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 import Carousel from './Carousel';
 import Base from './shared/Base';
 import { HeaderSections } from './shared/PlaynetConstants';
+import YouTube from './Youtube';
 
 function Home(): JSX.Element {
+  const [ chosenVideoPath, chooseVideo ] = useState(null);
+  const [ showCarousel, toggleCarousel ] = useState(false);
```

Comment:
nit: just for clarity's sake, I think it would be better to keep the convention that the second element of the array, which is the setter, is of the form set<Value> where <Value> is the name of the state variable. Like this:
```suggestion
  const [ chosenVideoPath, setChosenVideoPath ] = useState(null);
  const [ showCarousel, setShowCarousel ] = useState(false);
```
I think this also helps distinguish regular functions from state-modifying ones!